src/Application.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { ApolloProvider } from '@apollo/react-hooks';
+import { Helmet } from "react-helmet";
+
+import { config } from './config'
+import { client } from './Utils/apollo';
+import Router from './Router';
+
+const Application = () => (
+  <>
```

Comment:
Why you used empty bracket <> ?

Reply:
You can use React Fragment.

Reply:
I am glad that you learn new things. This is why I contribute to open-source projects.
<> is a shortcut for React Fragment (and do not need importing something)

Reply:
Oh Great 😀